server/src/test/java/org/opensearch/search/aggregations/bucket/ShardSizeTestCase.java

```diff
@@ -46,6 +46,7 @@
 import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertSearchResponse;
 import static org.hamcrest.Matchers.is;
 
+@OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST)
```

Comment:
This is not needed ? May be we can add in child classes if possible

Reply:
Sure, will update this